packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## 41.6.0 - October 25, 2017
+## 41.8.0 - Novemeber 16, 2017
+### Added
+
+#### salesforcedx-vscode-core
+
+* Option to run a single test class synchronously. *Requires v41.8.0 of the CLI* ([PR #206](https://github.com/forcedotcom/salesforcedx-vscode/pull/206))
+* SFDX: Create Project ([PR #197](https://github.com/forcedotcom/salesforcedx-vscode/pull/197))
+* SFDX: Refresh SObject Definitions ([Read more]())
```

Comment:
Will update link to Ruth's doc once that's ready.

Reply:
Will the doc be a readme in the salesforcedx-sobjects-faux-generator package? https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/packages/salesforcedx-sobjects-faux-generator

Reply:
I was going to add it to the salesforcedx-vscode-apex README. Do customers install salesforcedx-sobjects-faux-generator separately?

Reply:
No they can't install it directly. It's brought as a dependency through salesforcedx-vscode-core. I'm not sure if you want the README to be in core, or apex since the functionality is specific to apex.

Reply:
Hmm. It's much more related to the other content in the README for apex than it is for core. I'll add it in apex with a note that it requires core.